data/json/recipes/recipe_food.json

```diff
@@ -7693,6 +7693,18 @@
   ]
 },{
   "type" : "recipe",
+  "result": "vodka",
```

Comment:
What about lab grade ethanol? Or rubbing alcohol and similar low-quality sources?

This is not necessarily wrong, but it needs to be considered: do we want to drop the lab/food grade ethanol distinction?

If we assume all ethanol is food quality, then no problems here. But here we have a simplification: we have a simple chemical process (the food alcohol->ethanol recipe) that produces the ethanol.

I'd say it's more of a moonshine than vodka because of that, but I won't insist on changing it.